docs: add "first flight" docs for airspeed scale handling (#25987)
PX4/PX4-Autopilot: docs/en/config_fw/airspeed_scale_handling.md

> # Airspeed Scale Handling

Reviewer: FWIW, why "Handling"? YOu don't have to change this, but if you do, fix up the links too please

```suggestion
# Airspeed Scaling
```

- A scale of 1.0 means your sensor reads perfectly (no correction needed)
- A scale > 1.0 (e.g., 1.1) means your sensor _under-reads_ by 10%, so measured airspeed (IAS) must be multiplied by 1.1
- A scale < 1.0 (e.g., 0.9) means your sensor _over-reads_ by ~11%, so measured airspeed (IAS) must be multiplied by 0.9

### What Affects the Airspeed Scale

The primary factor influencing the airspeed scale is **sensor placement**.

Biased readings can be reflected in the scale estimate for pitot tubes installed:

- In turbulent regions
- Near propellers
- Under aerodynamic surfaces
- At an angle with respect to the airflow

### Symptoms of Incorrect Scale

Symptoms of an incorrectly scaled airspeed measurement include:

- Persistent under- or overestimation of the TAS relative to wind-corrected groundspeed
- False positives or missed detections in [airspeed innovation checks](../advanced_config/airspeed_validation.md#innovation-check)
- Degraded tracking of the rate controllers

## Recommended First Flight Process

During the first flight of a new fixed-wing vehicle, allocate time for the CAS scale to converge to a reasonable initial estimate.
Follow these steps:

> 1. **Set an Initial Scale**

Author: Noted  :heavy_check_mark:

# Airspeed Scale Handling

::: info
This section complements the existing [Airspeed Validation](../advanced_config/airspeed_validation.md) documentation.
:::

The airspeed scale is used by PX4 to convert the measured airspeed (indicated airspeed) to the calibrated airspeed.
This scale can be set by [ASPD_SCALE_n](../advanced_config/parameter_reference.md#ASPD_SCALE_1) (where `n` is the sensor number), and logged in [AirspeedWind.msg](../msg_docs/AirspeedWind.md).

> Note that the airspeed scale is different from the airspeed sensor offset calibration done on the ground at 0 m/s. The airspeed scale accounts for errors in the airspeed measurement during flight, such as those caused by sensor placement or installation effects.

Reviewer 2: ```suggestion
Note that the airspeed scale is different from the airspeed sensor offset calibration done on the ground without any measured airspeed. The airspeed scale accounts for errors in the airspeed measurement during flight, such as those caused by sensor placement or installation effects.
```

PX4 handles multiple types of airspeed:

- **IAS (Indicated Airspeed):** The raw measurement from the airspeed sensor, directly influenced by sensor characteristics and installation effects (e.g., pitot-static errors).
- **CAS (Calibrated Airspeed):** IAS corrected for sensor-specific and installation-related errors.
- **EAS (Equivalent Airspeed):** _Not explicitly handled by PX4_ - Calibrated airspeed corrected for compressibility effects.
  While PX4 does not currently model EAS separately, this correction is negligible at low speeds and altitudes, so EAS is treated as equivalent to CAS for simplicity.
- **TAS (True Airspeed):** CAS adjusted for atmospheric effects such as air pressure and temperature (i.e., altitude and atmospheric conditions).

The standard conversion chain used in PX4 is: `IAS → CAS (= EAS) → TAS`.

## CAS Scale Estimation

PX4 estimates the IAS to CAS scale (referred to as the CAS scale) during flight using GNSS ground speed and wind estimation.
To compute the final TAS, standard environment conversions are applied (CAS → TAS).

:::warning Important
A GNSS is required for scale estimation.
:::

PX4 uses a two-stage approach to robustly estimate the scale:

> 1. **Continuous EKF Estimation**: A wind estimator constantly compares your measured airspeed against what it expects based on ground velocity (from GPS) and estimated wind.

Reviewer 2: ```suggestion
1. **Continuous EKF Estimation**: A wind estimator constantly compares your measured airspeed against what it expects based on ground velocity (from GNSS) and estimated wind.
```
As you above already are using GNSS.
